Xamarin.Forms.Platform.iOS/Renderers/TabletFlyoutPageRenderer.cs

```diff
@@ -13,6 +14,11 @@ public override void ViewDidLayoutSubviews()
 			foreach (var vc in ChildViewControllers)
 				vc.View.Frame = View.Bounds;
 		}
+
+		public override UIViewController ChildViewControllerForStatusBarStyle()
+		{
+			return ChildViewControllers?.LastOrDefault() ?? base.ChildViewControllerForStatusBarStyle();
```

Comment:
ChildViewControllers is an array - can we just check the last index instead of allocating an enumerator?

Also, are we guaranteed that the last controller in the array is the correct one?

Reply:
Fixed it @hartez , this ChildVIewController wrapper is internal and we are the only ones adding to it the detail or flout as a children, it's the only children.